setup.sh fails on macOS ARM: R package 'fs' requires cmake

## Environment
- macOS 15.5 (Apple Silicon, arm64)
- R 4.x via renv
- cmake not pre-installed

## Problem
`bash setup.sh` fails during R package installation. The `fs` package (v2.0.0) builds libuv from source and requires `cmake`, which is not listed as a prerequisite.

```
/bin/sh: cmake: command not found
make: *** [libuv] Error 127
ERROR: compilation failed for package 'fs'
```

## Suggested Fix
1. Add `cmake` to prerequisites in README and ENVIRONMENT_QUICK_START.md
2. Add cmake check in `verify_environment.sh`
3. Or pin `fs` to an older version that doesn't require cmake

## Workaround
`brew install cmake` before running setup.sh

--- a/ENVIRONMENT_QUICK_START.md
+++ b/ENVIRONMENT_QUICK_START.md
@@ -26,6 +26,7 @@ bash verify_environment.sh
 - ✅ **Python 3.12+** - Data processing
 - ✅ **uv** - Python package manager (fast)
 - ✅ **R 4.3+** - Statistical analysis
+- ✅ **cmake** - Required for building R packages (e.g., `fs`) on macOS ARM
 - ⚠️ **JAGS** - Optional (only for Network Meta-Analysis)
 - ⚠️ **Quarto** - Optional (for manuscript rendering)
 
@@ -133,11 +134,12 @@ uv sync
 **macOS**:
 ```bash
 xcode-select --install
+brew install cmake  # Required for packages like 'fs' on ARM Macs
 ```
 
 **Linux**:
 ```bash
-sudo apt install build-essential libcurl4-openssl-dev
+sudo apt install build-essential libcurl4-openssl-dev cmake
 ```
 
 ### "JAGS not found"

--- a/README.md
+++ b/README.md
@@ -130,6 +130,7 @@ If you use meta-pipe in your research, please cite it:
 
 - `uv` (Python)
 - R ≥ 4.2 + `renv`
+- `cmake` (required for building R packages like `fs` on macOS ARM)
 - Quarto
 - API keys in `.env`
 

--- a/setup.sh
+++ b/setup.sh
@@ -74,6 +74,24 @@ echo "✅ Python environment ready"
 cd ../..
 echo ""
 
+# Step 3b: Check cmake (required for R packages like 'fs' on macOS ARM)
+echo "Checking cmake (needed for R package compilation)..."
+if ! command -v cmake &> /dev/null; then
+    echo "⚠️  cmake not found."
+    if [ "${PLATFORM}" = "macOS" ]; then
+        echo "   Install with: brew install cmake"
+    elif [ "${PLATFORM}" = "Linux" ]; then
+        echo "   Install with: sudo apt install cmake"
+    fi
+    echo "   cmake is required for building R packages like 'fs' on macOS ARM."
+    echo "   Please install cmake and run this script again."
+    exit 1
+else
+    CMAKE_VERSION=$(cmake --version | head -n1)
+    echo "✅ ${CMAKE_VERSION}"
+fi
+echo ""
+
 # Step 4: Check R
 echo "Step 4/5: Checking R..."
 if ! command -v R &> /dev/null; then

--- a/verify_environment.sh
+++ b/verify_environment.sh
@@ -35,6 +35,11 @@ check_command "python3" "python3 --version" || SYSTEM_OK=false
 check_command "uv" "uv --version" || SYSTEM_OK=false
 check_command "R" "R --version | head -n1" || SYSTEM_OK=false
 check_command "git" "git --version" || SYSTEM_OK=false
+check_command "cmake" "cmake --version" || {
+    echo -e "${YELLOW}⚠️  cmake is required for building some R packages (e.g., 'fs') on macOS ARM${NC}"
+    echo "   Install: brew install cmake"
+    SYSTEM_OK=false
+}
 
 # Optional tools
 if check_command "jags" "jags -v 2>&1 | head -n1"; then

